### DISCUSSION TITLE
Make license clear at the bottom of readme.

### DISCUSSION THREAD

**Discussion by elucian** [ykdojo/defaang#209] [Ideas]
I suggest you add a line using "---" at the bottom of readme file on a new line. Under it put your copyright:  like this:

---
Copyright (c) 2022 Yosuke Kyle Sugi

**Comment by ykdojo** [comment]
@elucian do we need a copyright notice when we have the MIT license for this repo?

**Reply by elucian** [reply]
Yes. Most licenses require you to display the license on the bottom of your artifact. For example on the website at the footer. Also on the readme on the footer. That's what I have read 10 years ago but read the license file to the bottom. MIT is a good license btw, but is used by organizations. Individuals use more GPL license or Apache licenses.

**Reply by ykdojo** [reply]
> That's what I have read 10 years ago but read the license file to the bottom.

Any recent reference article on this?
